src/managed/OpenLiveWriter.PostEditor/Video/VideoProvidersB2.xml

```diff
@@ -77,30 +77,20 @@
   <provider>
     <id>56744E0A-1892-4b56-B7FF-3A2568EE2D27</id>
     <serviceName>YouTube</serviceName>
-    <editorFormat xpp:if="version lt 15.4.3104.0816">
-      <![CDATA[
-		<embed src="http://www.youtube.com/v/{videoId}" type="application/x-shockwave-flash" wmode="transparent" width="{width}" height="{height}"></embed>
-		]]>
-    </editorFormat>
-    <editorFormat xpp:if="version gte 15.4.3104.0816">
+    <editorFormat>
       <![CDATA[
-		<embed src="http://www.youtube.com/v/{videoId}?hd=1" type="application/x-shockwave-flash" wmode="transparent" width="{width}" height="{height}"></embed>
+		<iframe src="http://www.youtube.com/embed/{videoId}" width="{width}" height="{height}" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 		]]>
     </editorFormat>
-    <embedFormat xpp:if="version lt 15.4.3104.0816">
-      <![CDATA[
-      <object width="{width}" height="{height}"><param name="movie" value="http://www.youtube.com/v/{videoId}&hl=en"></param><embed src="http://www.youtube.com/v/{videoId}&hl=en" type="application/x-shockwave-flash" width="{width}" height="{height}"></embed></object>
-		]]>
-    </embedFormat>
-    <embedFormat xpp:if="version gte 15.4.3104.0816">
+    <embedFormat>
       <![CDATA[
-      <object width="{width}" height="{height}"><param name="movie" value="http://www.youtube.com/v/{videoId}?hl=en&hd=1"></param><embed src="http://www.youtube.com/v/{videoId}?hl=en&hd=1" type="application/x-shockwave-flash" width="{width}" height="{height}"></embed></object>
+      <iframe src="http://www.youtube.com/embed/{videoId}&hl=en" width="{width}" height="{height}" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Seems the `&hl=en` is causing the player to error out (it's not valid URL parameter syntax). It can be changed to `?hl=en`, but I doubt the parameter is needed at all in this case. Removing it so the URL is just `http://www.youtube.com/embed/{videoId}` should be fine.

Reply:
I tried to follow the same syntax as in the other video providers, and I was trying with WordPress publishing. I agree that we should remove that query parameter. 👍